### PR TITLE
Update description in actions.yml

### PR DESCRIPTION
**Pull request by ethomson**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-name: 'Setup Dotnet environment'
-description: 'Setup a Dotnet environment and add it to the PATH, additionally providing proxy support'
+name: 'Setup .NET SDK'
```
Comment (ZEisinger) [inline action.yml]: Could we make it more clear that it is dotnet core? There were some questions about 4.5.* not installing in a previous issue.

Comment (rainersigwald) [reply action.yml]: I think the usual name used is `.NET Core SDK`, but our team's PM @kathleendollard probably has opinions here.

Comment (ethomson) [reply action.yml]: I think we can set up the framework SDKs as well, at least on Windows.

Comment (ethomson) [reply action.yml]: Does 4.5 *not work*?  It's documented.

Comment (ZEisinger) [reply action.yml]: It should not be documented and has been removed from the README in master. This is only for installing the .NET Core SDK.

Comment (ethomson) [reply action.yml]: Great, I updated it to say ".NET Core SDK".

**Pull request merged**